README.md

```diff
@@ -8,6 +8,11 @@ git clone https://github.com/balancer-labs/frontend-v2.git && cd frontend-v2
 ```
 
 ### Local env
+Copy .env.development to .env
+```bash
+cp .env.development .env
```

Comment:
>    Instead you seem to have used Container.get in these cases as well. Is there a reason that's required?

To my understanding the dependency only gets injected in constructors that are decorated with `@Service` themselves. If you were to move all the services to the dependency injection based system it would look a lot cleaner and you wouldn't need to use `Container.get(...)`.

Reply:
Perhaps it is a configuration issue on my end - I noticed it wouldn't succesfully run `npm run build` without a `.env`.